src/components/Search/Search.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import PropTypes from "prop-types";
+
+// actions
+import { searchGif } from "./../../actions";
+
+import UI from './Search.ui';
+
+class Search extends Component {
```

Comment:
Same thing, why are you using two components and one container? I still believe that you could just use one :).